lib/components/viewers/route-details.tsx

```diff
@@ -149,12 +149,12 @@ class RouteDetails extends Component<Props> {
             <HeadsignSelectLabel htmlFor="headsign-selector-label">
               <FormattedMessage id="components.RouteDetails.stopsTo" />
             </HeadsignSelectLabel>
-            <SortResultsDropdown
+            <Dropdown
```

Comment:
This is inheriting the text color of the page, which is white! 
<img width="474" alt="image" src="https://github.com/user-attachments/assets/4a5e942b-fa16-48dd-9fab-c3051832aeef">

Reply:
Good catch!